code/player/TTTPlayer.Spectating.cs

```diff
@@ -83,5 +84,30 @@ public void MakeSpectator(bool useRagdollCamera = true)
 
             ShowFlashlight(false, false);
         }
+
+        public void ForceSpectator()
```

Comment:
since we do `IsForcedSpectator = !IsForcedSpectator;` this sounds like a `ToggleForceSpectator` instead of `ForceSpectator`.

Reply:
Agreed. Will make change